docstring/src/test/java/io/cucumber/docstring/DocStringTest.java

```diff
@@ -5,20 +5,9 @@
 
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class DocStringTest {
 
-    @Test
-    void throws_when_no_converter_defined() {
```

Comment:
This test still seems correct. Why was it removed?

Reply:
I thought it's better suited in DocStringTypeRegistryDocStringConverterTest, but you're right it doesn't belong there, I will revert this.